cmd/crc/cmd/podman_env.go

```diff
@@ -14,36 +10,17 @@ var podmanEnvCmd = &cobra.Command{
 	Short: "Setup podman environment",
 	Long:  `Setup environment for 'podman' binary to access podman on CRC VM`,
 	Run: func(cmd *cobra.Command, args []string) {
-
-		// See issue #961; Currently does not work on Windows in combination with the CRC vm.
-		errors.ExitWithMessage(1, "Currently not supported.")
-
-		userShell, err := shell.GetShell(forceShell)
-		if err != nil {
-			errors.ExitWithMessage(1, "Error running the podman-env command: %s", err.Error())
-		}
-
-		ipConfig := machine.IpConfig{
-			Name:  constants.DefaultName,
-			Debug: isDebugLog(),
-		}
-
-		exitIfMachineMissing(ipConfig.Name)
-
-		result, err := machine.Ip(ipConfig)
-		if err != nil {
-			errors.Exit(1)
+		if err := runPodmanEnv(args); err != nil {
+			errors.ExitWithMessage(1, err.Error())
 		}
-
-		output.Outln(shell.GetPathEnvString(userShell, constants.CrcBinDir))
```

Comment:
This is relevant and will be restored soon.

Reply:
I will argue this is why we use git. When it will be ready, we can found it really quickly in the history and restore it.
Also, the linter doesn't like dead code and it refuses to build. Here, the error was hidden by in os.Exit before.